### PR TITLE
Add a lot of things.

### PR DESCRIPTION
**Pull request by RhythmsAreFromVibrations**
``.refresh`` as an alias for ``.reload`` for a different way to reload the bot.
``.bannedphrases``, ``.bannedwords``, and ``.bw`` as aliases for ``.viewbannedwords`` to allow more ways to see the banned words.
Added "No." to ``.8ball`` for another guaranteed answer.
``.thp`` and ``.happyplace`` as aliases for ``.happy`` to allow newer users who only know The Happy Place as "thp" or "happyplace."

### CODE REVIEW

**Comment by RhythmsAreFromVibrations** [comment]
``friendcode`` and ``fc`` cases because they're other ways of checking someone's FC :o

**Comment by Morfent** [comment]
.8ball's based off the responses from an 8ball. http://en.wikipedia.org/wiki/Magic_8-Ball#Possible_answers
I'm iffy on .reload and .vbw's aliases being terribly useful, but .thp and .thehappyplace are good aliases for .happy.

**Comment by RhythmsAreFromVibrations** [comment]
So... remove the aliases added for .checkfc and "no." for .8ball?

**Comment by Morfent** [comment]
As well as .reload and .vbw.

**Comment by codelegend** [comment]
@RhythmsAreFromVibrations , squash the commits. (use `git rebase -i` for the interactive rebase)
Also, give a proper commit summary ("A lot of things" can mean anything)

File: commands.js [hunk 1]
```diff
@@ -341,7 +341,6 @@ exports.commands = {
 		this.writeSettings();
 		this.say(con, room, "Phrase \"" + arg + "\" is no longer banned.");
 	},
-	viewbannedphrases: 'viewbannedwords',
```
Comment (Morfent) [inline commands.js]: Why are you removing this?

**Comment by Morfent** [comment]
It also helps if you write a comment after adding new commits or rebasing/amending, since I don't get any notification otherwise.

**Comment by RhythmsAreFromVibrations** [comment]
Sorry :/

I'll re-add vbw and then I'll squash

**Comment by codelegend** [comment]
any updates @RhythmsAreFromVibrations ?

**Comment by RhythmsAreFromVibrations** [comment]
lol woops forgot about this

**Comment by codelegend** [comment]
bump

**Comment by Morfent** [comment]
At this rate, I'll just add the aliases myself. Closing this.

**Comment by RhythmsAreFromVibrations** [comment]
Thanks. I've been pretty busy, so I keep forgetting about this.

**Pull request closed without merging**